File: Week1_Friday/UsingSwift/UsingSwift/ViewController.m
//
//  ViewController.m
//  UsingSwift
//
//  Created by trainer on 1/4/16.
//  Copyright © 2016 trainer. All rights reserved.
//

#import "ViewController.h"
#import "UsingSwift-Swift.h"
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    MyClass *myClass = [[MyClass alloc] init];
    [myClass greeting:@"hello"];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: Week2_Friday/UserDefaults/UserDefaults/ViewController.swift
//
//  ViewController.swift
//  UserDefaults
//
//  Created by kongyunpeng on 1/14/16.
//  Copyright © 2016 kongyunpeng. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
     let defaults = NSUserDefaults.standardUserDefaults()
     let myDefaults = ["greeting":"hello","numberOfItems":1]
     // defaults.registerDefaults(myDefaults)
        
    print(defaults.objectForKey("greeting"))
    defaults.setObject(NSDate(), forKey:"date")
    print(defaults.objectForKey("date"))
    
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

